modules/i3s/src/lib/parsers/parse-i3s-tile-content.js

```diff
@@ -36,17 +36,22 @@ export async function parseI3STileContent(arrayBuffer, tile, tileset, options) {
   tile.content.attributes = {};
 
   if (tile.textureUrl) {
-    const url = getUrlWithToken(tile.textureUrl, options.token);
-    const loader = FORMAT_LOADER_MAP[tile.textureFormat] || ImageLoader;
-    tile.content.texture = await load(url, loader);
-    if (loader === CompressedTextureLoader) {
-      tile.content.texture = {
-        compressed: true,
-        mipmaps: false,
-        width: tile.content.texture[0].width,
-        height: tile.content.texture[0].height,
-        data: tile.content.texture
-      };
+    const uvChecker = options.i3s.uvChecker;
```

Comment:
I think we should not change the loader here. The loader should always load the right texture. We should override with the debug texture in the display stage.

Reply:
It should be deck.gl then

Reply:
Well, I am not sure that having support for debug textures in deck.gl makes too much sense as a general feature, just like it doesn't make much sense in the loader. 

I think one option is that the the debug application could do this. It could for instance just update the textures in the loaded tiles before display. Store the original texture in userData and swap in the debug texture.